Writerside/topics/Git-Strategy.md

```diff
@@ -0,0 +1,108 @@
+# Git Strategy
+
+GitFlow is a workflow that defines a strict branching model designed around the project release. It assigns very
```

Comment:
Are we sure we want to work with gitflow? I feel like a trunc-based approach would be leaner: No develop branch, merge short-living feature branches directly into master. Lets discuss at the workshop.

Reply:
i just added that sample from the mermaid docs. Lets build our own process in a workshop.